src/main/resources/lang/de_de.json

```
@@ -0,0 +1,121 @@
+{
+	"language": "Deutsch",
+
+	"mapping_format.enigma_file": "Enigma‐Datei",
+	"mapping_format.enigma_directory": "Enigma‐Verzeichnis",
+	"mapping_format.enigma_zip": "Enigma‐ZIP‐Archiv",
+	"mapping_format.tiny_v2": "Tiny‐v2‐Datei",
+	"mapping_format.tiny_file": "Tiny‐Datei",
+	"mapping_format.srg_file": "SRG‐Datei",
+	"mapping_format.proguard": "ProGuard‐Datei",
+	"type.methods": "Methoden",
+	"type.fields": "Felder",
+	"type.parameters": "Parameter",
+	"type.classes": "Klassen",
+
+	"menu.file": "Datei",
+	"menu.file.jar.open": "Java‐Archiv öffnen …",
+	"menu.file.jar.close": "Java‐Archiv schließen",
+	"menu.file.mappings.open": "Zuordnungen laden …",
+	"menu.file.mappings.save": "Zuordnungen speichern",
+	"menu.file.mappings.save_as": "Zuordnungen speichern unter …",
```

```suggestion
	"menu.file.mappings.save_as": "Zuordnungen exportieren …",
```
To match the English

---

That string is already a direct translation from the English language setting:
```"menu.file.mappings.open": "Open Mappings...",
"menu.file.mappings.save": "Save Mappings",
"menu.file.mappings.save_as": "Save Mappings As...",
"menu.file.mappings.close": "Close Mappings",
"menu.file.mappings.drop": "Drop Invalid Mappings",

---

https://github.com/FabricMC/Enigma/blob/1708c4cf6c6e79144ac06e45bf0d4a67e779d8a5/src/main/resources/lang/en_us.json#L21

---

Hm, where did I see export then